uwblueprint/childrens-aid-society
# Setup Pages For Intake Form

## Notion ticket link

[Setup Pages For Intake Form](https://www.notion.so/uwblueprintexecs/Setup-pages-for-intake-form-46b41368174a4705bf39946c772074a9)



## Implementation description
* Added button functionality so can move back and forth on intake pages



## Steps to test
1. Load on localhost:3000, and visit localhost:3000/intake
2. Click on the previous/next buttons to see if navigate from (Case referral, Court Information, Individual Details, Program details)


## What should reviewers focus on?
* If navigation works between the intake pages


## Checklist
